lib/v2/design/app_theme.dart

```diff
@@ -19,6 +19,7 @@ class SeedsAppTheme {
         behavior: SnackBarBehavior.floating,
         contentTextStyle: TextStyle(color: AppColors.white),
       ),
+      indicatorColor: AppColors.green1,
```

Comment:
I add this to the other theme too. Not sure if there is a color for this but to keep this aligned with the other.